### ISSUE TITLE
A problem of multiclassifier task with Squeezenet trained on VOC2012

### ISSUE THREAD

**Issue by Lollipop9z** [pytorch/vision#2329]
I got a problem when I dealed with a multiclassifier task with squeezenent on VOC2012. I just wrote a train code, and called the '''torchversion.models.squeezenet1_1''', changed num_classes. I used '''torch.nn.MultiLabelSoftMarginLoss()''' for my loss function. However, my loss never changed when I trained my network. If there is someone having same problem like me, and having some specific soluation, please help me. please! Thank you~
```
Epoch: [  0/2000] step:  0, Loss: 0.754, mAP 26.93%
Epoch: [  0/2000] step: 20, Loss: 0.693, mAP 7.48%
Epoch: [  0/2000] step: 40, Loss: 0.693, mAP 6.65%
Epoch: [  0/2000] step: 60, Loss: 0.693, mAP 6.43%
Epoch: [  0/2000] step: 80, Loss: 0.693, mAP 6.39%
Epoch: [  0/2000] step: 100, Loss: 0.693, mAP 6.55%
Epoch: [  0/2000] step: 120, Loss: 0.693, mAP 6.83%
```

**Comment by pmeier** [comment]
Hi @Lollipop9z, please note that this issue tracker is not a help form and this issue will be closed. Our primary means of support is our [discussion forum](https://discuss.pytorch.org/).

**Comment by fmassa** [comment]
Closing the issue following @pmeier comment.
